File: tuan2/PhanA/bai33.jl
function main()
    s = strip(read(stdin, String))
    isempty(s) && return
    n = parse(Int, s)

    count = 0
    sum_val = 0

    while n > 0
        sum_val += n % 10
        count += 1
        n = div(n, 10)
    end

    println("$count $sum_val")
end

main()

File: tuan2/PhanA/bai33.py
import sys

def main():
    s = sys.stdin.read().strip()
    if not s: return
    n = int(s)
    
    count = 0
    total_sum = 0
    
    while n > 0:
        total_sum += n % 10
        count += 1
        n = n // 10
        
    print(f"{count} {total_sum}")

if __name__ == "__main__":
    main()